packages/aws-cdk-lib/aws-ses/lib/configuration-set.ts

```diff
@@ -15,6 +15,13 @@ export interface IConfigurationSet extends IResource {
    * @attribute
    */
   readonly configurationSetName: string;
+
+  /**
+   * The ARN of the configuration set
+   *
+   * @attribute
+   */
+  readonly configurationSetArn: string;
```

Comment:
The `@attribute` might not be appropriate since it's derived from the resource ref, a maintainer should weigh in on this

Reply:
In case it helps, the exemple construct from the coding guideline uses `@attribute` for the ARN property even if it is computed manually (like in this case) :

- `exempleResourceArn` is tagged with `@attribute`

https://github.com/aws/aws-cdk/blob/6fdc4582f659549021a64a4d676fce12fc241715/packages/%40aws-cdk/example-construct-library/lib/example-resource.ts#L70-L83

- `exempleResourceArn` is constructed manually in `from` static methods and constructor
https://github.com/aws/aws-cdk/blob/6fdc4582f659549021a64a4d676fce12fc241715/packages/%40aws-cdk/example-construct-library/lib/example-resource.ts#L362-L368


The only doubt I have is regarding this precision : `It needs to be annotated with '@attribute' if the underlying CloudFormation resource surfaces the ARN as a return value.`
CloudFormation does not surface `AWS::SES::ConfigurationSet` ARN.